css/style.css

```diff
@@ -6,6 +6,110 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+svg {
+    padding: 20px 0 20px 0;
+    height: 50px;
+}
+
+.navbar li{
+    display: inline;
+    padding: 0 10px 0 40px;
+}
+
+a {
+    text-decoration: none;
+    list-style: none;
+    color:black;
+    font-size: 10px;
+}
+
+a:hover {
+    color: rgb(255, 102, 0);
+}
+
+header nav {
+    float:right;
+    margin-top: 10px;
+}
+
+.social {
+    height: 50px;
+}
+
+button {
+    background-color: rgb(255, 102, 0);
+    padding: 10px 10px 10px 10px;
+    border-style: hidden;
+    color:white;
+    align-content: center;
+}
+
+#coverpage {
```

Comment:
Your page has horizontal scroll which is not very good if you use flex in your containers then you can manage it better